hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/traceability/TraceabilitySuite.java

```diff
@@ -299,7 +300,7 @@ private HapiSpec traceabilityE2EScenario1() {
                                                 .setCallType(CREATE)
                                                 .setCallOperationType(CallOperationType.OP_CREATE)
                                                 .setCallingAccount(TxnUtils.asId(GENESIS, spec))
-                                                .setGas(197000)
+                                                .setGas(48248)
```

Comment:
Some of these gas numbers are _smaller_ than before - so that's not the effect of accounting for input bytes.  What is it?